docs/pages/concepts/networking/components/network-manager.md

```diff
@@ -1,21 +1,98 @@
-4. The CRDReplicator running in Cluster1 reflects this changes in the local resource `NCFG(1->2)`. Same step done in Cluster2; The TunnelEndpointCreator in Cluster1 combining the status of `NCFG(1->2)` and the spec of `NCFG (2->1)` it has all the necessary information to create the custom resource `TEP (1-2)` of type `tunnelendpoints.net.liqo.io`.
+1. The network configuration of Cluster1 is saved in a local custom resource (`NCFG(1->2)`) containing the ClusterID of the remote cluster to whom is destined. Same step done in Cluster2;
+2. The `CRDReplicator` running in Cluster1 (which knows how to interact with the API server of Cluster2) replicates `NCFG(1->2)` in the remote cluster. The same step is done in Cluster2;
+3. The custom resource `NCFG(1->2)` in Cluster2 is processed and its status updated with the NAT information (if any). The same is done with `NCFG(2->1)` in Cluster1.
+4. The `CRDReplicator` running in Cluster1 reflects this changes in the local resource `NCFG(1->2)`. Same step done in Cluster2; The TunnelEndpointCreator in Cluster1 combining the status of `NCFG(1->2)` and the spec of `NCFG(2->1)` it has all the necessary information to create the custom resource `TEP(1-2)` of type `tunnelendpoints.net.liqo.io`.
+
+
+### IPAM
+Embedded within the Liqo Network Manager, the IPAM (IP Address Management) is the Liqo module in charge of:
+1. Manage networks and IP addresses, as the name suggests.
+2. Translate IP addresses of offloaded Pods (assigned by foreign clusters) in addresses valid on the home cluster.
+3. Provide IP addresses for endpoints to the Virtual Kubelet during the [Reflection](../../../offloading/features/api-reflection#Overview).
+
+
+#### Networks and addresses management
+Liqo enables Pods in different (peered) clusters to talk one to another.
+Communicating over the Internet requires involved entities to have an IP address; this is not a problem for Pods, as they are assigned an address by the CNI installed on their cluster.
+However, these IPs are valid only within the context of the cluster and therefore are not reacheable by Pods in peered clusters.
+The [Tunnel Operator](../gateway#tunnel-operator) solves this problem, enabling Pods to communicate not only with those running on their same cluster, but also with those running on peered clusters. 
+
+But how does the IPAM come into the rescue?
+Liqo enables traffic toward clusters' PodCIDR but does not make any assumption on those address spaces.
+It is legitimate to imagine a situation in which two peered clusters have the same PodCIDR: in this case two Pods could be assigned the same IP addresses, creating a conflict.
+
+{{% notice info %}}
+The PodCIDR is the network used by the CNI for assigning addresses to Pods.
+The ExternalCIDR is the network used by Liqo for assigning addresses to external resources during Reflection(See more [Here](#reflection-of-external-endpoints)).
```

Comment:
```suggestion
The ExternalCIDR is the IP addressing space used by Liqo for assigning addresses to external resources during Reflection (See more [Here](#reflection-of-external-endpoints)).
```

Reply:
1) The ExternalCIDR comes here out of the blue, it has never been presented.
2) Not clear what this IP addressing space is. Please improve the definition.